File: 18_05_2024_Find_the_Highest_number.cpp
class Solution {
public:
    
    int findPeakElement(vector<int>& a)
    {
        int max = a[0];
        int i;
        for (i = 1; i < a.size(); i++) {
            if (a[i] > max)
                max = a[i];

            
            else
                break;
        }

        return max;
    }
};

File: 18_05_2024_Find_the_Highest_number.java
class Solution {
    public int findPeakElement(List<Integer> a)
    {
        // Code here
        int max = a.get(0);
        int i;
        for (i = 0; i < a.size(); i++) {
            if (a.get(i) > max)
                max = a.get(i);
        }
        return max;
    }
}
